routes/users.js

```diff
@@ -267,4 +268,6 @@ router.post("/picture", authenticate, upload.single("profile"), users.postUserPi
 
 router.patch("/identityURL", authenticate, userValidator.updateIdentityURL, users.identityURL);
 
+router.patch("/:username", authenticate, authorizeUser("superUser"), users.updateUser);
```

Comment:
It's to make sure that their profile data is moderated. Superuser get's list of all profile changes(limited) and depending on the data it can be rejected or approved.

Reply:
This api is called after the profile changes is approved.